src/extensions/default/CodeSnippetsAdder/README.md

```diff
@@ -0,0 +1 @@
+# CodeSnippetsAdder
```

Comment:
This file isn't overly helpful in its current form, so we should either remove it, or add to it.